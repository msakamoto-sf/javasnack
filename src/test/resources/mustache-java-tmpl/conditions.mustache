{{#cond1}}
item1={{item1}}
{{/cond1}}
{{^cond1}}
empty(item1)
{{/cond1}}
{{#cond2}}item2={{item2}}{{/cond2}}{{^cond2}}empty(item2){{/cond2}}
{{#cond3}}item3={{item3}}{{/cond3}}{{^cond3}}empty(item3){{/cond3}}
{{#cond4}}item4={{item4}}{{/cond4}}{{^cond4}}empty(item4){{/cond4}}
{{#cond5}}item5={{item5}}{{/cond5}}{{^cond5}}empty(item5){{/cond5}}
