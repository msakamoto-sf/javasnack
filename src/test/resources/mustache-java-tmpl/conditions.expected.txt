item1=hello
empty(item2)
empty(item3)
empty(item4)
empty(item5)
